src/views/settings/tabs/RecommendedCapabilitiesTab/utils/constants.ts

```
@@ -0,0 +1,166 @@
+    description: t(
+      'Configure and maintain consistent host networking configuration across cluster nodes using NMState.',
+    ),
+    id: 'host-network-management',
+    isDefaultBundle: true,
+    operators: [{ packageName: NMSTATE_OPERATOR_NAME }],
+    title: t('Host network management'),
+  },
+  {
+    description: t(
+      'Run workloads in lightweight VMs using Kata Containers for enhanced security.',
+    ),
```

ESLint

[error] Replace `⏎······'Run·workloads·in·lightweight·VMs·using·Kata·Containers·for·enhanced·security.',⏎····` with `'Run·workloads·in·lightweight·VMs·using·Kata·Containers·for·enhanced·security.'`

(prettier/prettier)